### PR TITLE
Acceleration and owner in wire protocol

### PR DESCRIPTION
**Pull request by sethalves**
These values aren't yet used, this PR just makes room for them in the wire-protocol.

### CODE REVIEW

**Comment by hifi-gustavo** [comment]
Build console output for commit: 51a0fdf3d46b8372866b276117b9fe024ec4ee82 available at: https://gist.github.com/a4e5ab36ba39c86252ff<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4617.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4617.dmg

**Comment by murillodigital** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 51a0fdf3d46b8372866b276117b9fe024ec4ee82 available at: https://gist.github.com/a4e5ab36ba39c86252ff<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4617.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4617.dmg

File: libraries/entities/src/EntityItemProperties.h [hunk 1]
```diff
@@ -95,6 +95,8 @@ enum EntityPropertyList {
     
     PROP_COLLISION_MODEL_URL,
     PROP_MARKETPLACE_ID,
+    PROP_ACCELERATION,
+    PROP_SIMULATOR_ID,
     
     ////////////////////////////////////////////////////////////////////////////////////////////////////
     // ATTENTION: add new properties ABOVE this line and then modify PROP_LAST_ITEM below
```
Comment (AndrewMeadows) [inline libraries/entities/src/EntityItemProperties.h]: As per the comment, you forgot to modify PROP_LAST_ITEM.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 62ff8bf7357ed58e774129f500a64d60feb22118 available at: https://gist.github.com/a4e5ab36ba39c86252ff<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4617.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4617.dmg

**Comment by ZappoMan** [comment]
Do we have a published design document on this concept of simulation node? I'd really like to understand our plan for using this property. For example what happens when the simulation node goes away and the ID is no longer valid?

**Comment by sethalves** [comment]
We don't have one.  The current goal is to "fix popcorn".  It looks like including acceleration in the protocol and having the entity-server ignore updates that don't match its current idea of the owner will take care of that.  Our (Andrew, Philip, myself) conversation about it seemed to indicate that the entity-server was going to have to be smarter about who the simulation owner is, but we haven't solved ping-pong yet.

**Comment by PhilipRosedale** [comment]
I will write up a google design doc capturing our design conversation thusfar and post here.

**Comment by ZappoMan** [comment]
I strongly encourage us to not go down this road. I believe moving the authoritative simulation taak to the responsibility of an AC based node is a superior solution. Scaling can be more easily maintained by muliple ACs divided by spatial or populatin round robin. Ownership handoff is no longer a problem in this design. Popcorning is eliminated because there's only one editor.

**Comment by PhilipRosedale** [comment]
@ZappoMan:  Hang on, let me write something up, I think you are missing the idea here.  Whether there is an authoritative AC or not, clients will (at least sometimes) need to 'Edit' objects while interacting with them, and this PR is to give the entity server and others the information on who 'owns' the simulation of an object.  So, for example, in the case where objects were near multiple simulating AC's you would still need this code.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 73a126562468ed3de859ae8e4c90cfde44db87bf available at: https://gist.github.com/a4e5ab36ba39c86252ff<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4617.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4617.dmg

**Comment by murillodigital** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 73a126562468ed3de859ae8e4c90cfde44db87bf available at: https://gist.github.com/a4e5ab36ba39c86252ff<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4617.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4617.dmg

**Comment by murillodigital** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 73a126562468ed3de859ae8e4c90cfde44db87bf available at: https://gist.github.com/a4e5ab36ba39c86252ff<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4617.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4617.dmg

**Comment by murillodigital** [comment]
test this please

**Comment by PhilipRosedale** [comment]
Here is the to the design document:  
https://docs.google.com/document/d/1gPVwc43ApGJGtZOnHud7kFvMa0fxemEJBL1TPER48eo/edit?usp=sharing

@ZappoMan Let us know if you are OK with us trying to get this working, and if you have time to review the PR.

**Comment by hifi-gustavo** [comment]
Build console output for commit: 73a126562468ed3de859ae8e4c90cfde44db87bf available at: https://gist.github.com/a4e5ab36ba39c86252ff<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4617.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4617.dmg

**Comment by murillodigital** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 73a126562468ed3de859ae8e4c90cfde44db87bf available at: https://gist.github.com/a4e5ab36ba39c86252ff<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4617.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4617.dmg

**Comment by murillodigital** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 73a126562468ed3de859ae8e4c90cfde44db87bf available at: https://gist.github.com/a4e5ab36ba39c86252ff<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4617.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4617.dmg

**Comment by murillodigital** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 73a126562468ed3de859ae8e4c90cfde44db87bf available at: https://gist.github.com/a4e5ab36ba39c86252ff<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4617.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4617.dmg

**Comment by murillodigital** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 73a126562468ed3de859ae8e4c90cfde44db87bf available at: https://gist.github.com/a4e5ab36ba39c86252ff<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4617.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4617.dmg

**Comment by murillodigital** [comment]
test this please

**Comment by hifi-gustavo** [comment]
Build console output for commit: 73a126562468ed3de859ae8e4c90cfde44db87bf available at: https://gist.github.com/a4e5ab36ba39c86252ff<br/><br/>Installers available for download at: <br/>Windows: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4617.exe<br/>OS X: http://s3.amazonaws.com/hifi-public/pr-builds/dev-interface-4617.dmg

**Pull request merged**